nexus-sdk/src/main/java/io/nexusrpc/OperationState.java

```diff
@@ -9,5 +9,5 @@ public enum OperationState {
   /** Indicates an operation completed as failed. */
   FAILED,
   /** Indicates an operation completed as canceled. */
-  CANCELLED,
+  CANCELED,
```

Comment:
@cretz @bergundy I had to change this because the spelling here matters since it is part of the spec https://github.com/nexus-rpc/api/blob/main/SPEC.md#operationinfo

I didn't go around and change all the other occurrences of of `canceled` , but I can if we want to be consistent

Reply:
FWIW, I think we should do as the languages prefer if there is any preference. At least this is how I approached things with `CancelledError` in Python vs `CanceledError` in .NET and Ruby though our API uses `CanceledFailureInfo`.

Reply:
In Java I would say `CANCELLED` is more common but the string representation of this enum needs to be `canceled` so are you OK with the string representation of this enum being different or are you saying you are OK with the enum being inconsistent with the rest of the SDK?

Reply:
I personally prefer matching SDK/lang over matching API here, but I also understand that Nexus SDK != Temporal SDK, so we don't have to follow the Temporal SDK.

Reply:
So the enum string representation **has** to be `canceled` per the Nexus spec

Reply:
Yes, it has to be `canceled`, I think we should keep the enum value consistent to prevent confusion.

Reply:
User-facing SDK API isn't in spec, and of course we can translate as needed (we already have to with casing). I do think it will be a bit more confusing in, say, Python to have "canceled" for Nexus high-level API only but "cancelled" in every other hihg-level API. I think we should leave the option open for these other languages to be idiomatic to their language more than the spec.

Reply:
Yeah I am not making an comment on other languages, all I am saying is per the Nexus spec the operation state is called `canceled` and I am respecting that here in this enum.